lib/mutant/repository.rb

```diff
@@ -20,17 +20,9 @@ def call(subject)
 
     # Diff between two objects in repository
     class Diff
-      include Adamantium, Concord.new(:from, :to)
+      include Adamantium, Anima.new(:config, :from, :to)
```

Comment:
The config object is a pure value distribution object. It should not know how to interpret the configured values. The reminder of the Mutant namespace is the "evaluator for the config".